Drawer - Update 'Left or Right Position' demo description (#1814)
DevExpress/devextreme-demos: JSDemos/Demos/Drawer/LeftOrRightPosition/description.md

> Drawer is a dismissible or permanently visible panel that usually holds navigation elements. 

Reviewer: ```suggestion
Drawer is a dismissible or permanently visible panel that usually contains navigation elements. 
```

> Drawer is a dismissible or permanently visible panel that users can use for navigation in applications. 

Reviewer 2: The phrase "users can use" prompted the change. Phrases don't read well if they contain similar words in close proximity. 

```suggestion
Drawer is a dismissible or permanently visible panel that usually holds navigation elements. 
```

> In this demo, when users click on a toolbar item, it opens the Drawer component. The component can interact with the view in the following modes (see [openedStateMode](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#openedStateMode)): 

Reviewer 2: A few reasons why I wanted to edit: 

- The first sentence refers to "users" - some third party... But we can (and should) speak directly to the reader in demo description. They are the developers who came here to try these demos. So we can instruct them what to do. 
- The first sentence contains a couple of comma-separated clauses. 
- "it" is a bit vague. Who opens the drawer? The "item" opens? Or the action of the "click" opens?
- "a toolbar item" sounds like "any toolbar item". Before I looked at the demo, I assumed I would find several buttons and I can click any of them. Definite article ("the") would be the proper choice in this case. I can even be specific and say "the 'Hamburger' button". (And it's not that important if it's part of a toolbar.)
- "interact with the view" - is too generic. Once again, I wanted to be a bit more specific and mention that actions affect the layout in different ways.

```suggestion
This demo displays the 'Hamburger' button that opens or closes the Drawer component. Each Drawer visibility change can adjust the layout in the following ways (see [openedStateMode](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#openedStateMode)): 
```

or 

```suggestion
To open or close the Drawer component in this demo, click the 'Hamburger' button. Each Drawer visibility change can adjust the layout in the following ways (see [openedStateMode](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#openedStateMode)): 
```

- "*push*"    
The Drawer partially displaces the view.

- "*shrink*"    

> The view's width decreases to accommodate the Drawer.

Reviewer 2: Not a big deal, but I would maybe just say "shrinks" instead of "width decreases". Seems more straightforward, more natural. 

```suggestion
The view shrinks to accommodate the Drawer.
```

- "*overlap*"    
The Drawer overlaps the view.

> In this demo, you can use radio buttons under the Drawer to configure this property. You can also change Drawer's [position](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#position) and [revealMode](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#revealMode) properties.

Reviewer 2: I think we already established that we now use the context of the demo, so I wouldn't start another sentence with "In this demo".

```suggestion
Use the radio buttons at the bottom of this demo module to try different layout modes. You can also change the Drawer's [position](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#position) and [revealMode](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#revealMode) properties.
```

> Use a [template](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#template) to define the Drawer's content. The view's content (an HTML string in this demo) is part of the Drawer's markup. Note that the Drawer component does not interact with the toolbar. The code of the demo does not define the toolbar in the Drawer's markup.

Reviewer 2: So you talk about three different things in this paragraph: drawer's template, how view content is inside the drawer markup, and that the toolbar is on its own. I would then write the first sentence that somehow unites all this information. In general, every paragraph's first sentence defines the main idea. Then you proceed to specifics. 

```suggestion
Note this demo's markup. The Drawer definition encloses the view content (the formatted text). The Drawer's own content (navigation links) is set by the [template](/Documentation/ApiReference/UI_Components/dxDrawer/Configuration/#template) property. Finally, the toolbar markup is separate from other elements.
```
